article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -28,6 +29,11 @@
         @if(model.currentPage.currentPage.isArchivePage) {
             @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage, amp = true)
         }
+
+            <script type="application/ld+json">
```

Comment:
@NataliaLKB @gtrufitt The coding for this turned out to be a bit messy - since Metadata is in common, it doesn't have access to the data structures used to decide the range of blocks to present on the page that are coded into the article module, and I don't really want to duplicate the code if I can help it